openfe/protocols/openmm_utils/system_creation.py

```diff
@@ -87,10 +84,11 @@ def get_system_generator(
         'cutoffnonperiodic': app.CutoffNonPeriodic,
         'cutoffperiodic': app.CutoffPeriodic,
         'ewald': app.Ewald
-    }[system_settings.nonbonded_method.lower()]
+    }[forcefield_settings.nonbonded_method.lower()]
 
-    nonbonded_cutoff = to_openmm(
-        system_settings.nonbonded_cutoff
+    nonbonded_cutoff = ensure_quantity(
```

Comment:
why the switch between to ensure_quantity from to_openmm? I'm pretty sure last I looked the code for ensure_quantity just used to_openmm in the background